### ISSUE TITLE
Word order backwards; examples are wrong

### ISSUE THREAD

**Issue by hyperrealgopher** [muesli4/table-layout#14]
```
> import Text.Layout.Table.Justify 
Text.Layout.Table.Justify> justifyText 10 "This text will not fit on one line."
["This","will  text","on fit not","line. one"]
```

This is the example from the docs. I am using version `0.9.0.1` and GHC version `8.8.4`. It seems to have the reverse ordering of words.

**Comment by muesli4** [comment]
This bug was introduced in e482677a4e2e9f019f95fc76337d53e01636a954.  I will provide a fix today.

**Comment by muesli4** [comment]
I released version `0.9.0.2` on hackage that contains the fix.  Thanks for reporting the bug!
